build/devenv/tests/e2e/smoke_test.go

```diff
@@ -101,9 +104,9 @@ func TestE2ESmoke(t *testing.T) {
 					OutOfOrderExecution: true,
 				})
 				require.NoError(t, err)
-				_, err = c.WaitOneSentEventBySeqNo(ctx, tc.fromSelector, tc.toSelector, seqNo, 1*time.Minute)
+				_, err = c.WaitOneSentEventBySeqNo(ctx, tc.fromSelector, tc.toSelector, seqNo, defaultSentTimeout)
 				require.NoError(t, err)
-				e, err := c.WaitOneExecEventBySeqNo(ctx, tc.fromSelector, tc.toSelector, seqNo, 5*time.Minute)
+				e, err := c.WaitOneExecEventBySeqNo(ctx, tc.fromSelector, tc.toSelector, seqNo, defaultExecTimeout)
```

Comment:
This timeout was way too long before anyway. When tests pass the longest I see here is 25s. In many cases it passes in 5s.